scripts/_docker.sh

```diff
@@ -23,14 +23,16 @@ case $PUBLISH in
 
 esac
 
-exec docker buildx build \
+docker buildx build \
   --platform=linux/amd64 \
   --cache-from=type=local,src=$PROJECT_ROOT/cache \
+  --cache-from=type=local,src=$PROJECT_ROOT/cache_output \
   -f ${DIR}/Dockerfile \
   --target=$TARGET $PUBLISH_TO_REGISTRY \
-  --build-arg BUILDKIT_INLINE_CACHE=1  \
   --build-arg APP=${APP} \
   --build-arg APP_HOME=${APP_HOME} \
   --build-arg APP_DIST_HOME=${APP_DIST_HOME} \
   -t ${DOCKER_REGISTRY}${APP}:${DOCKER_TAG} \
   $PROJECT_ROOT
+
+docker inspect ${DOCKER_REGISTRY}${APP}:${DOCKER_TAG} | jq ".[].RootFS" 
```

Comment:
this was just for debugging right? remove?

Reply:
on it

Reply:
fixed